### PR TITLE
Added an accessibility question and fixed minor issue in security-quiz

### PR DESCRIPTION
**Pull request by Rishav9911**
## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

File: src/data/accessibility-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,14 @@
 const accessibilityQuiz = [
+
+  {
+    Question: "What does WCAG (sometimes pronounced wee-kag) stand for?",
```
Comment (Gautam-Hegde) [inline src/data/accessibility-quiz.ts:4]: This question is redundant since it already exists

Comment (Rishav9911) [reply src/data/accessibility-quiz.ts:4]: I am sorry i did not see that it already exists , can I add another one?

Comment (jdwilkin4) [reply src/data/accessibility-quiz.ts:4]: @Rishav9911 

Yes, you can come up with another question if you like

Comment (Gautam-Hegde) [reply src/data/accessibility-quiz.ts:4]: Certainly! Feel free to proceed, just ensure there are no syntax errors.

**Pull request closed without merging**